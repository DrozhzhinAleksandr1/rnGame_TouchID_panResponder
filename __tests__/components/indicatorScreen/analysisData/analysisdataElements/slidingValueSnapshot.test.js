import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { SlidingValue } from '../../../../../app/components/indicatorScreen/analysisData/analysisDataElements/slidingValues';

it('create snapshot SlidingValue', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<SlidingValue text={'R1'} number={0.946587} />);
    expect(toolbarSnapshot).toMatchSnapshot();
});
