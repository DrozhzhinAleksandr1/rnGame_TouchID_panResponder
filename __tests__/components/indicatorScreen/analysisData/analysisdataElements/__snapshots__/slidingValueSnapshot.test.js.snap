// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot SlidingValue 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Roboto-Light",
        "fontSize": 14,
        "lineHeight": 14,
        "marginRight": 8,
        "marginTop": 3,
      }
    }
  >
    R1
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Roboto-Regular",
        "fontSize": 14,
        "lineHeight": 16,
      }
    }
  >
    0.946587
  </Text>
</Component>
`;
